### PR TITLE
pkp/pkp-lib#7171 Simplify setup of Jobs page

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: pages/management/PKPToolsHandler.inc.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 use APP\i18n\AppLocale;
 use APP\notification\NotificationManager;
 use APP\template\TemplateManager;
-use PKP\components\PKPStatsJobsTable;
+use Illuminate\Support\Facades\Queue;
```
Comment (henriqueramos) [inline pages/management/PKPToolsHandler.inc.php:25]: This is being used?

File: templates/management/tools/jobs.tpl [hunk 2]
```diff
@@ -10,20 +10,15 @@
...
-            </pkp-header>
-            <tabs label="{translate key="navigation.tools.jobs"}">
-                <tab label="{translate key="navigation.tools.queuedJobs"}" id="showQueuedJobs" badge="{{$totalQueuedJobs}}">
-                    <jobs-table
-                        :table_data=components.queuedJobsTable
-                    />
-                </tab>
-            </tabs>
-        </div>
+	<h1 class="app__pageHeading">
```
Comment (henriqueramos) [inline templates/management/tools/jobs.tpl:13]: Tabs instead spaces?

File: pages/management/PKPToolsHandler.inc.php (hunk 1 above)
Comment (NateWr) [reply pages/management/PKPToolsHandler.inc.php:25]: Sorry, test code that I forgot to remove before committing.  Can you remove  it after merging?

File: templates/management/tools/jobs.tpl (hunk 2 above)
Comment (NateWr) [reply templates/management/tools/jobs.tpl:13]: Yes, on smarty templates.

**Review by henriqueramos — APPROVED** [review]

**Pull request merged**